components/switch/index.css

```diff
@@ -10,38 +10,70 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/scales/spectrum-large.css";
-@import "../vars/css/scales/spectrum-medium.css";
-@import "../vars/css/components/spectrum-switch.css";
 
-.spectrum-Switch {
-  @remapvars {
-    find: /--spectrum-switch-m(.*)/;
-    filter: color;
-    replace: --spectrum-switch$1;
-  }
 
-  /* Hardcoded for wrapping study.
-  Will be a DNA token in https://jira.corp.adobe.com/browse/SDS-4466 */
-  --spectrum-switch-label-margin-top: var(--spectrum-global-dimension-size-65);
+.spectrum-Switch-sizeS {
+  --spectrum-switch-height: var(--spectrum-component-height-75);
+  --spectrum-switch-control-width: var(--spectrum-switch-control-width-small);
+  --spectrum-switch-control-height: var(--spectrum-switch-control-height-small);
+  --spectrum-switch-control-label-spacing: var(--spectrum-text-to-control-75);
+  --spectrum-switch-spacing-top-to-control: var(--spectrum-switch-top-to-control-small);
+  --spectrum-switch-spacing-top-to-label: var(--spectrum-component-top-to-text-75);
+
+  --spectrum-switch-font-size: var(--spectrum-font-size-75);
+}
+
+/* @TODO add support for lang CJK-line-height */
```

Comment:
Will there be any adjustments for CJK support? Are we still waiting on design/the core tokens for this?